src/index.js

```diff
@@ -24,33 +21,19 @@ app.use('/api-docs', swaggerUi.serve, swaggerUi.setup(swaggerDocument));
 dotEnv.config();
 // Normal express config defaults
 app.use(morgan('dev'));
-app.use(bodyParser.urlencoded({ extended: false }));
-app.use(bodyParser.json());
+app.use(express.urlencoded({ extended: false }));
+app.use(express.json());
 
-app.use(methodOverride());
 app.use(express.static(`${__dirname}/public`));
 
-app.use(
-  session({
-    secret: 'authorshaven',
-    cookie: { maxAge: 60000 },
-    resave: false,
-    saveUninitialized: false,
-  }),
-);
-
 if (!isProduction) {
   app.use(errorHandler());
 }
 
-app.use(routes);
+app.use('/api/v1', routes);
```

Comment:
Why was this remove from the routes folder and brought here

Reply:
This is just due to preference. They both do the same thing

Reply:
When it is merged with mine which resides in routes folder  won't that result to conflict

Reply:
No it won't. U can put yours in the folder and remove this from the index file